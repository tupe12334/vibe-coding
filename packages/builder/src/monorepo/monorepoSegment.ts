import type { RootContent } from "mdast";

export const monorepoSegment = async (
  system: string
): Promise<RootContent[]> => {
  const segment: RootContent[] = [
    {
      type: "paragraph",
      children: [
        {
          type: "text",
          value: `Monorepo system: ${system}`,
        },
      ],
    },
  ];

  return segment;
};
