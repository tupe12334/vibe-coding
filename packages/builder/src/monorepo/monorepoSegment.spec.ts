import { describe, expect, test } from "vitest";
import { monorepoSegment } from "./monorepoSegment";

describe("monorepoSegment", () => {
  test("returns monorepo system line", async () => {
    const segment = await monorepoSegment("turbo");
    expect(segment).toMatchSnapshot();
  });
});
