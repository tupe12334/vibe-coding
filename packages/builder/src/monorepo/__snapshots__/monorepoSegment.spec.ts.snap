// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`monorepoSegment > returns monorepo system line 1`] = `
[
  {
    "children": [
      {
        "type": "text",
        "value": "Monorepo system: turbo",
      },
    ],
    "type": "paragraph",
  },
]
`;
